lettermgmt/models/letter_channel.py

```diff
@@ -1,33 +1,16 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    Parthiv Pate, Tech Receptives, Open Source For Ideas
-#    Copyright (C) 2009-Today Tech Receptives(http://techreceptives.com).
-#    All Rights Reserved
-#
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU General Public License as published by
-#    the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU General Public License for more details.
-#
-#    You should have received a copy of the GNU General Public License
-#    along with this program.  If not, see http://www.gnu.org/licenses/.
-#
-##############################################################################
-from openerp.osv import fields, orm
+# -*- coding: utf-8 -*-
+# Copyright (C) 2009-Today Tech Receptives(http://techreceptives.com).
+# Parthiv Pate, Tech Receptives, Open Source For Ideas
+# Hardikgiri Goswami <hardikgiri.goswami@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+from openerp import models, fields
 
-class letter_channel(orm.Model):
+
+class LetterChannel(models.Model):
     """ Class to define various channels using which letters can be sent or
     received like : post, fax, email. """
     _name = 'letter.channel'
     _description = "Send/Receive channel"
-    _columns = {
-        'name': fields.char('Type', required=True),
-    }
+
+    name = fields.Char('Type', required=True)
```

Comment:
Maybe remove field string `'Type'`, since the field is supposed to be the name of the letter channel, just `'Name'` makes sense.